chrome/content/zotero/xpcom/attachments.js

```diff
@@ -2339,6 +2339,24 @@ Zotero.Attachments = new function () {
 			return value;
 		};
 
+		const relationalOperators = (value, { isGreaterThan = null, isGreaterThanOrEqual = null, isLessThan = null, isLessThanOrEqual = null } = {}) => {
```

Comment:
"Equal" → "EqualTo"

Reply:
Fixed!

But thinking about this again, maybe it would be better to modify the template engine to allow relational operators so the syntax would look like this:  

`{{ if authorsCount >= 2 }}`  

The reason for this is that it's confusing to use `==` for equality while relying on descriptive arguments for inequality.  

Instead, I could extend template engine here:

https://github.com/zotero/zotero/blob/7a467b8bfa24ebd78bedbce389aa9678a8150914/chrome/content/zotero/xpcom/utilities_internal.js#L2197-L2203

and we would document this feature only for numbers. (For strings, it would follow JavaScript's default behaviour of comparing strings alphabetically, but we would not document that.)

I've now convinced myself that this would be a better approach. Thoughts?